packages/dart/sshnoports/bundles/universal.sh

```diff
@@ -755,6 +755,11 @@ client() {
   "$extract_path"/sshnp/install.sh -b "$bin_path" -u "$user" srv
   "$extract_path"/sshnp/install.sh -b "$bin_path" -u "$user" at_activate
 
+  # set PATH so it includes user's private bin if it exists
+  if [ -d "$user_home/.local/bin" ] ; then
+    PATH="$user_home/.local/bin:$PATH"
```

Comment:
Don't we need export here?

Reply:
no need for export as it's within the subshell for the script that we need this.